dist/angular-validation.js

```diff
@@ -387,7 +388,7 @@
       if (element.attr('no-validation-message')) {
         messageElem.css('display', 'none');
       } else if ($validationProvider.showSuccessMessage && messageToShow) {
-        messageElem.html($validationProvider.getSuccessHTML(messageToShow));
+        messageElem.html('').append($compile($validationProvider.getSuccessHTML(messageToShow))(scope));
```

Comment:
Should it be `messageElem.html().append($compile($validationProvider.getSuccessHTML(messageToShow))(scope));`?